python/foglamp/common/storage_client/storage_client.py

```diff
@@ -311,6 +312,97 @@ def disconnect(self):
 
         return jdoc
 
+    async def post_snapshot(self, tbl_name):
+        """Create a table snapshot
+
+        :param tbl_name:
+        :return:
+
+        :Example:
+            curl -X POST http://0.0.0.0:8080/storage/table/configuration/snapshot
+        """
+        post_url = '/storage/table/{tbl_name}/snapshot'.format(tbl_name=tbl_name)
+        data = {"id": str(int(time.time()))}
+
+        url = 'http://' + self.base_url + post_url
+        async with aiohttp.ClientSession() as session:
+            async with session.post(url, data=json.dumps(data)) as resp:
+                status_code = resp.status
+                jdoc = await resp.text()
+                if status_code not in range(200, 209):
+                    _LOGGER.info("POST %s", post_url)
```

Comment:
If `post_url` info is required, then it should be before this condition or if its for debug purpose either remove it or with .debug. Same comments applied to all endpoints

Reply:
In my opinion this is okay as-is. Only in the case of error, we would prefer to see (info) what url was called and what error is there, even in production system.